### ISSUE TITLE
QKV LoRA also works on noise and text branches?

### ISSUE THREAD

**Issue by laolongboy** [Xiaojiu-z/EasyControl#71]
Quoted from paper 3.1, 
> By applying LoRA-based adaptation exclusively to the
Condition Branch, we ensure that conditional signals are efficiently injected into the model without disrupting the pretrained text and noise representations

And the Figure 2 also emphasize that the LoRA of qkv only influence with the condition branch.

<img width="339" alt="Image" src="https://github.com/user-attachments/assets/0c3e4065-22c2-4900-b02d-2ad2abaab66d" />

But I found the noise tokens and condition tokens concat directly and going to the same attention module, so how to make LoRA only influenced condition branch? 

[norm_hidden_states = torch.concat([norm_hidden_states, norm_cond_hidden_states], dim=-2)](https://github.com/Xiaojiu-z/EasyControl/blob/351ff8278e7a7f1fe4ba7bd98814d1bea401ef06/train/src/transformer_flux.py#L173-L183)

**Comment by laolongboy** [comment]
In addition, I did not find the attention mask injection required for causal attention. How to implement the complete mechanism described in the paper?
